src/audio/module_adapter/module/waves.c

```diff
@@ -433,15 +411,13 @@ static int waves_effect_buffers(struct processing_module *mod)
 
 	waves_codec->i_buffer = i_buffer;
 	waves_codec->o_buffer = o_buffer;
-	waves_codec->response = response;
 	codec->mpd.in_buff = waves_codec->i_buffer;
 	codec->mpd.in_buff_size = waves_codec->buffer_bytes;
 	codec->mpd.out_buff = waves_codec->o_buffer;
 	codec->mpd.out_buff_size = waves_codec->buffer_bytes;
 
-	comp_info(dev, "waves_effect_buffers() size response %d, i_buffer %d, o_buffer %d",
-		  waves_codec->response_max_bytes, waves_codec->buffer_bytes,
-		  waves_codec->buffer_bytes);
+	comp_info(dev, "waves_effect_buffers() size i_buffer %d, o_buffer %d",
+		  waves_codec->buffer_bytes, waves_codec->buffer_bytes);
```

Comment:
It would be better to use codec->mpd.in_buff_size, codec->mpd.out_buff_size here.